powerline/bindings/lemonbar/powerline-lemonbar.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import os
+import sys
+import time
+
+from threading import Lock, Timer
+from argparse import ArgumentParser
+
+from powerline.lemonbar import LemonbarPowerline
+
+
+if __name__ == '__main__':
+	parser = ArgumentParser(description='Powerline lemonbar bindings.')
+	parser.add_argument(
+		'--i3', action='store_true',
+		help='Subscribe for i3 events.'
+	)
+	parser.add_argument(
+		'--width', action='store', type=int,
+		default=None, help='Maximum width the bar can occupy'
+	)
+	parser.add_argument(
+		'theme', default=None,
+		help='The theme to render')
```

Comment:
Same thing: use overrides from environment if you are sure you need this at all.